pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/impl/BrokersBase.java

```diff
@@ -317,7 +318,7 @@ public void healthcheck(@Suspended AsyncResponse asyncResponse,
                             pulsar().getConfiguration());
 
 
-            topic = String.format("persistent://%s/healthcheck", heartbeatNamespace);
+            topic = String.format("persistent://%s/%s", heartbeatNamespace, HEALTH_CHECK_TOPIC_SUFFIX);
```

Comment:
You may  Don’t need format

Reply:
Could you give me more information to help me know why i don't need format :)